NHS.Api/requirement27.txt

```diff
@@ -0,0 +1,6 @@
+basho-erlastic==2.1.1
```

Comment:
As we moved dependency to setup , so if this file is not required, remove or make a comment on top of it

Reply:
I have kept requirement.txt file as a backup type, if in any case setup.py fails then requirement.txt file will serve the purpose.